ArjunaCore/arjuna/pom.xml

```diff
@@ -422,7 +422,7 @@
           <systemPath>${orson.jar.location}/orson-0.5.0.jar</systemPath>
         </dependency>
         <dependency>
-          <groupId>jfree</groupId>
+          <groupId>org.jfree</groupId>
```

Comment:
As far as I know the org.jfree is the new location of the artifact (since 2009).  There are new features and some bug fixed in the version 1.5.3. Not many changes even if more then 10 years passed from the older version.